### PR TITLE
feat(sandbox): example of casbin as a provider

### LINKED ISSUE #148: Need example of casbin
**Is your feature request related to a problem? Please describe.**
Right now we dont have any complete example of casbin provider of authorization

**Describe the solution you'd like**
Add example, documentation and blog

**Describe alternatives you've considered**
None.

**Additional context**
None

### PR DESCRIPTION
**Pull request by ashutosh-bansal-2136**
## Description

Example of casbin as a provider using loopback4-authorization

Fixes #148 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [x] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [x] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,58 @@
...
 The current catalog consists of the following services:
 
-* [audit-service](services/audit-service)
-* [authentication-service](services/authentication-service)
-* [in-mail-service](services/in-mail-service)
-* [notification-service](services/notification-service)
-* [scheduler-service](services/scheduler-service)
-* [video-conferencing-service](services/video-conferencing-service)
-* [bpmn-service](services/bpmn-service)
+- [audit-service](services/audit-service)
```
Comment (samarpan-b) [inline README.md:12]: why these changes ?

File: sandbox/casbin-example/src/providers/casbin-enforce-config.provider.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  ): Promise<CasbinConfig> {
+    if (isCasbinPolicy) {
+      const model = path.resolve(
+        __dirname,
+        './../../fixtures/casbin/model.conf',
+      );
+      // Write business logic to find out the allowed resource-permission sets for this user. Below is a dummy value.
+      const allowedRes = [{resource: 'user', permission: 'TodoCRUD'}];
+
+      const policy = path.resolve(
```
Comment (samarpan-b) [inline sandbox/casbin-example/src/providers/casbin-enforce-config.provider.ts:40]: reading policy from a file is not recommended. we need to do this from DB. @mlakdawala @mlakdawala-sf @sf-kansara can one of you please guide ?

**Review by samarpan-b — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ashutosh-bansal-2136) [reply README.md:12]: These were not added in readme
